script/workLoggingSystem.js

```diff
@@ -135,8 +139,9 @@ $("#submitNewTask").click(function() {
     console.log(estimatedTime);
     console.log(status);
     console.log(category);
+    console.log(loggingDate);
 
-    createNewTask(title,des,priority,difficulty,creator,assignee,estimatedTime,status,category);
+    createNewTask(title,des,priority,difficulty,creator,assignee,estimatedTime,status,category,loggingDate);
```

Comment:
better to call creationDate rather than loggingDate

Reply:
sure, I will do.